.vitepress/config.ts

```diff
@@ -256,7 +256,7 @@ export const sidebar: ThemeConfig['sidebar'] = {
       ]
     },
     {
-      text: 'Scaling Up',
+      text: 'Mise à l\'échelle',
```

Comment:
Pas fan de la traduction.

Scaling up fait référence à montée en charge dans un environnement d'infrastructures informatique.
Mais là c'est plutôt une histoire de monter en difficulté d'apprentissage.

Du coup je dirai plutôt :


```suggestion
      text: 'Approfondir',
```

Reply:
Pas de soucis merci pour ta proposition j'ai eu des doutes concernant le bon terme je voulais aussi avoir votre avis sur le terme le mieux adapté Approfondir.